<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="1.186" timestamp="2023-06-23T12:32:31.759155" hostname="DESKTOP-KP2HK5D"><testcase classname="" name="gitactionstest.test.test_get_api" time="0.000"><error message="collection failure">gitactionstest\test\test_get_api.py:19: in &lt;module&gt;
    test_get_method()
gitactionstest\test\test_get_api.py:17: in test_get_method
    assert response.status_code == 201, f"Status code showing wrong"
E   AssertionError: Status code showing wrong
E   assert 200 == 201
E    +  where 200 = &lt;Response [200]&gt;.status_code</error></testcase></testsuite></testsuites>